### ISSUE TITLE
SCSS support

### ISSUE THREAD

**Issue by jkomyno** [EQuimper/thefreelancetools#1]
Would you be interested in adding SCSS support, with automatic watch-and-build?
I could send a PR for this issue.

**Comment by EQuimper** [comment]
Yes sure, I would use it :). Not a lot cause I plan to use primitive or styled-components but for some layout yes :)
